plaintext/README.md

```diff
@@ -0,0 +1,146 @@
+# Plaintext Connection Protocol
+
+> An insecure connection handshake for non-production environments.
+
+| Lifecycle Stage | Maturity      | Status | Latest Revision |
+|-----------------|---------------|--------|-----------------|
+| 1A              | Working Draft | Active | r0, 2019-05-27  |
+
+
+Authors: [@yusefnapora]
+
+Interest Group: TBD
+
+[@yusefnapora]: https://github.com/yusefnapora
+
+See the [lifecycle document][lifecycle-spec] for context about maturity level
+and spec status.
+
+[lifecycle-spec]: https://github.com/libp2p/specs/blob/master/00-framework-01-spec-lifecycle.md
+
+## Overview
+
+Secure communications are a key feature of libp2p, and encrypted transport is
+configured by default in libp2p implementations to encourage security for all
+production traffic. However, there are some use cases such as testing in which
+encryption is unnecessary. For such cases, the plaintext "security" protocol can
+be used. By conforming to the same interface as real security adapters like
+[SECIO][secio-spec] and [TLS][tls-spec], the plaintext module can be used as a
+drop-in replacement when encryption is not needed.
+
+As the name suggests, the plaintext security module does no encryption, and all
+data is transmitted in plain text. However, peer identity in libp2p is [derived
+from public keys][peer-id-spec], even when peers are communicating over an
+insecure channel. For this reason, peers using the plaintext protocol still
+exchange public keys and peer ids when connecting to each other.
+
+This document describes the exchange of peer ids and keys that occurs when
+initiating a plaintext connection. This exchange happens after the plaintext
+protocol has been negotiated as part of the [connection upgrade
+process][conn-spec-conn-upgrade].
+
+## Protocol Id and Version History
+
+The plaintext protocol described in this document has the protocol id of
+`/plaintext/1.1.0`. 
```

Comment:
Since this is a breaking change, we should probably bump the version up to 2.0.0.